examples/13. Named Endpoints.md

```diff
@@ -0,0 +1,41 @@
+FORMAT: 1A
+
+# Named Endpoints API
+This API example demonstrates how to define a standalone endpoint with an identifier.
+
+## API Blueprint
++ [Previous: Advanced Action](12.%20Advanced%20Action.md)
++ [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/13.%20Named%20Endpoints.md)
+
+# Group Quick start
+
+## Create user [POST /users]
+
+The first thing we need to do is to add a user that we can later log in as. Do
+this by sending a POST request with the relevant details.
+
++ Request
+  + Headers
```

Comment:
The second-level list items should be indented by 4 spaces according to our API Blueprint examples style guide.